Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -69,10 +69,18 @@ section FixedPoints
 
 variable {M α}
 
+@[to_additive] theorem mem_fixedPoints_iff_subsingleton_orbit {a : α} :
+    a ∈ fixedPoints M α ↔ (orbit M a).Subsingleton := by
```

Comment:
What do you think of turning this around and making it simp?

Reply:
Why not, although it is not in par with the next lemma (that uses Fintype.card).